components/search_engines/brave_prepopulated_engines.h

```diff
@@ -6,24 +6,28 @@
 #ifndef BRAVE_COMPONENTS_SEARCH_ENGINES_BRAVE_PREPOPULATED_ENGINES_H_
 #define BRAVE_COMPONENTS_SEARCH_ENGINES_BRAVE_PREPOPULATED_ENGINES_H_
 
-#include <cstddef>
 #include <map>
 
 #include "build/build_config.h"
 #include "third_party/search_engines_data/resources/definitions/prepopulated_engines.h"
 
 namespace TemplateURLPrepopulateData {
 
-// IMPORTANT! Make sure to bump this value if you make changes to the
-// engines below or add/remove engines.
-inline constexpr int kBraveCurrentDataVersion = 32;
+// ****************************************************************************
+// IMPORTANT! Make sure to bump the value of kBraveCurrentDataVersion in
```

Comment:
is there some way to know when this breaks? Can we static_assert or something?

Reply:
`LINT.IfChange`?

Reply:
Would errors show up in presubmit?

Reply:
When I try adding those and then making a change inside one I don't see any errors when building or running presubmit. They are still helpful visually, so I can just check that in, but would be nice to know what they are supposed to trigger.

Reply:
Added in https://github.com/brave/brave-core/pull/28471/commits/44a1e0b921140c36ab93030971ee47a5dcfd16cc

Reply:
I'm not sure what triggers it, but I see them in upstream. @cdesouza-chromium do you know?

Reply:
fyi https://github.com/ebrevdo/ifttt-lint `An open‑source implementation of Google's internal IfThisThenThat (IFTTT) linter tool` @cdesouza-chromium is looking into it